stl/inc/type_traits

```diff
@@ -1877,8 +1934,8 @@ public:
     }
 
     template <class... _Types>
-    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), _STD forward<_Types>(_Args)...)) {
-        return _STD invoke(get(), _STD forward<_Types>(_Args)...);
+    auto operator()(_Types&&... _Args) const -> decltype(_STD invoke(get(), static_cast<_Types&&>(_Args)...)) {

```

Comment:
Shouldn't this be `_CONSTEXPR20`?

Reply:
I think this change implements most/all of #51; consider finishing it.

Reply:
After refreshing my memory of WG21-P1065, there are a few product changes / a lot of test changes necessary to complete it; I retract this comment. You got me excited with the `_CONSTEXPR20` on `invoke`.